deep_conmech/simulator/matrices/matrices_2d.py

```diff
@@ -5,12 +5,11 @@
 import numpy as np
 from conmech.features.boundaries import Boundaries
 from conmech.helpers import nph
-from deep_conmech.simulator.setting.setting_mesh import SettingMesh
 from numba import njit
 
 
 @njit  # (parallel=True)
-def get_edges_features_matrix_2d_numba(elements, nodes):
+def get_edges_features_matrix_numba(elements, nodes):
     ELEMENT_NODES_COUNT = 3
```

Comment:
Can you move these constants outside the function?

Reply:
Done